spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -174,6 +174,28 @@ def batch
     RUBY
   end
 
+  it 'corrects `EnforcedStyle: hash_rockets` of `Style/HashSyntax` with `Layout/HashAlignment`' do
+    create_file('.rubocop.yml', <<~YAML)
+      Style/HashSyntax:
+        EnforcedStyle: hash_rockets
+    YAML
+    source = <<~RUBY
+      some_method(a: 'abc', b: 'abc',
+              c: 'abc', d: 'abc'
+              )
+    RUBY
+    create_file('example.rb', source)
+    expect(cli.run([
+                     '--auto-correct-all',
+                     '--only', 'Style/HashSyntax,Style/HashAlignment'
+                   ])).to eq(0)
+    expect(IO.read('example.rb')).to eq(<<~RUBY)
+      some_method(:a => 'abc', :b => 'abc',
+                  :c => 'abc', :d => 'abc'
+              )
+    RUBY
+  end
+
```

Comment:
Amazing - I just spent about four hours trying to get this to work via way overcomplicating the `expect_correction` method (basically by adding a `let(:run_first)` variable). This is a million times cleaner. :) Going to miss having a PR to Rubocop but I guess I'll have other opportunities!

Reply:
I'm curious: what was the difficulty with `expect_correction`? This test with the CLI is much slower...

Reply:
The issue is that you can only run one cop at a time. There's no easy way to pass it multiple cops to run one after another. This bug only happens when one cop makes a change and the next one makes a second change - the two insertions end up in an invalid state.

Reply:
I was under the impression that `let(:other_cops) { { 'Other/Cop' => ... } }` was the way to go, but indeed looks like it's not the case. I opened #8352 as it should be easily doable to specify additional cops.